demos/classical_ct_headneck_affine/README.md

```diff
@@ -54,6 +54,23 @@ warped image/labels with the ground-truth image/labels.
 python demos/classical_ct_headneck_affine/demo_register.py
 ```
 
+## Visualise
+
+The following command can be executed to generate a plot of three image slices from the
+the fixed image, moving image and warped image (left to right) to visualise the
```

Comment:
sorry being picky, a better order would be moving, warped and fixed, such that the warped can be compared with the other two, see how much changed and how close to the target fixed.

Reply:
Good point, I'll make this change and update the images once the unit test problem is resolved